treeherder/model/models.py

```diff
@@ -615,6 +615,7 @@ class JobDetail(models.Model):
 
     class Meta:
         db_table = "job_detail"
+        unique_together = ["title", "value", "job"]
```

Comment:
I think ("title", "value", "job") syntax gives a better hint that this is a single group, though there is no functional difference.